beacon-chain/blockchain/service.go

```diff
@@ -471,6 +471,7 @@ func (s *Service) removeStartupState() {
 // It returns the (potentially updated) custody group count and the earliest available slot.
 func (s *Service) updateCustodyInfoInDB(slot primitives.Slot) (primitives.Slot, uint64, error) {
 	isSubscribedToAllDataSubnets := flags.Get().SubscribeAllDataSubnets
```

Comment:
While we are here, could we rename `SubscribeAllDataSubnets` to `Supernode`? (And `isSubscribedToAllDataSubnets` to `isSemiSuperNode`)?

Reply:
https://github.com/OffchainLabs/prysm/pull/16029/commits/700894ef9d18b2f66f0f21052d8b7fb319e8512b addressed here